include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -2993,14 +2994,45 @@ namespace internal
                     .component_dof_indices_offset[active_fe_index]
                                                  [first_selected_component];
 
+                const unsigned int n_filled_lanes =
+                  dof_info.n_vectorization_lanes_filled[dof_access_index][cell];
+
+                const bool vectorization_possible =
+                  (n_face_orientations == 1) &&
+                  (n_filled_lanes == VectorizedArrayType::size());
```

Comment:
This check has to be generalized with an additional `&&` checking if all cells are locally owned.